core-bundle/src/Twig/Studio/Operation/SaveOperation.php

```diff
@@ -41,6 +41,7 @@ public function execute(Request $request, OperationContext $context): Response
             return $this->error($context);
         }
 
+        // FIXME: Use ->query, ->request or ->attributes
```

Comment:
The parameter is set in the body for the fetch: https://github.com/contao/contao/blob/5.x/core-bundle/assets/controllers/template-studio-controller.js#L85-L88

So, it is `->request` then, I guess.